worlds/noita/options.py

```diff
@@ -118,7 +118,7 @@ class ShopPrice(Choice):
 
 class NoitaDeathLink(DeathLink):
     """
-    When you die, everyone dies. Of course, the reverse is true too.
+    When you die, everyone who enabled death link dies. Of course, the reverse is true too.
```

Comment:
```suggestion
    __doc__ = DeathLink.__doc__ + "\n\n    You can disable this or set it to give yourself a trap effect when another player dies in the in-game mod options."
```
Stealing Berserker's fix and using this opportunity to update it slightly.